lib/wda/xcodebuild.js

```diff
@@ -16,6 +16,7 @@ const BUILD_TEST_DELAY = 1000;
 
 const DERIVED_DATA_FOLDER_REGEXP = /(\/.+\/DerivedData\/WebDriverAgent-[^\/]+)/;
 const DERIVED_DATA_LOG_REGEXP = /\s+(\/.+\/WebDriverAgentRunner-.+\/.+\.log)/;
+const DERIVED_DATA_GREP_REGEXP = '/WebDriverAgentRunner-';
```

Comment:
If this is called a regexp it ought to be a regexp.

Reply:
it's not a real regular expression object, since it's just used as a grep argument. I've renamed the constant